### ISSUE TITLE
What tools should we include on the Tools and programs page in docs?

### ISSUE THREAD

**Issue by samccann** [ansible-community/community-topics#220]
### Summary

We have the following page in docs - https://docs.ansible.com/ansible/latest/community/other_tools_and_programs.html

Which lists things like editors that have ansible extensions, and then we have another list of tools etc. Today, we haven't set a guideline on what is included on that page. 

So, should we : 

1. Only list projects that the community is maintaining (things like vscode plugins and antsibull-docs etc)
2. List projects that  community members are directly involved in, even though not covered as part of the Ansible community directly.
3. List projects that the community verifies are useful and well-maintained.
4. List anything anyone opens a PR to add :-)

**Comment by samccann** [comment]
> > Only list projects that the community is maintaining (things like vscode plugins and antsibull-docs etc)
> 
> How exactly is this defined? Should these be projects that live in one of the ansible GH organizations (gh.com/ansible-community, etc.)? Or are (some) projects outside of these orgs (or even outside GitHub) also covered?

If we say 'community-maintained only' then yes, I think it would live in one of the Ansible github orgs.

**Comment by gotmax23** [comment]
I agree with `#2` or `#3`. I feel queasy about limiting this to certain Github organizations. Projects shouldn't be required to use Github or be under a specific umbrella. For sure, projects should be related to ansible, open source, useful to the community at large, and maintained.

**Comment by samccann** [comment]
Community WG feedback:

General guideline - use https://github.com/ansible-community/awesome-ansible as the full curated list. Use this page only for the tools the community maintains. Anything valid we remove from this docs page should be added to awesome ansible.

Other feedback:
- return the list to alphabetical order
 add pycharm link
- remove Jetbrains as not actively maintained
- add toc to top of page
- add disclaimer to all tools etc that aren't maintained by the community (vscode, ara, etc)
- update link to awesome ansible to -https://github.com/ansible-community/awesome-ansible

**Comment by samccann** [comment]
Also remove https://ansible.sivel.net/pr/byfile.html and check which Jetbrains is still active vs https://github.com/MSDehghan/AnsiblePlugin/issues/22

**Comment by felixfontein** [comment]
I like the suggestion to simply point to https://github.com/ansible-community/awesome-ansible instead of having another list of stuff that needs to be curated / updated.

Also we should contact the creator of https://github.com/jdauphant/awesome-ansible whether they want to collaborate on https://github.com/ansible-community/awesome-ansible instead of having a separate list.

**Comment by ssbarnea** [comment]
awesome ansible is awesome but I noted that is manually made, see https://github.com/ansible-community/awesome-ansible/issues/86
